fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/meeting/CreateMeeting.java

```diff
@@ -29,9 +32,14 @@ public class CreateMeeting extends Data {
    * @throws IllegalArgumentException if the id is invalid
    */
   public CreateMeeting(
-      String laoId, String id, String name, long creation, String location, long start, long end) {
-    if (!id.equals(
-        Hash.hash(EventType.MEETING.getSuffix(), laoId, Long.toString(creation), name))) {
+      String laoId,
+      String id,
+      String name,
+      long creation,
```

Comment:
You could use `MessageValidator` to check that the times are valdid :)

Reply:
I used the messageValidator to verify the ids, however to check if start and end are valid the current checkValidOrderedTimes doesn't work, as it checks also that the end is not in the future while it could. So depending on which PR gets merged first then we can modify that accordingly

Reply:
Ah yes you are right.